db/migrations/0015_add_customer_deleted.sql

```diff
@@ -0,0 +1,2 @@
+alter table stripe.customers
+    add deleted boolean default false not null;
```

Comment:
should we do a deleted_at? since we also have an updated_at column

Reply:
there is no deleted_at provided. https://stripe.com/docs/api/customers/delete